recipe/conda_build_config.yaml

```diff
@@ -1,10 +1,19 @@
 blas_impl:
   - mkl                        # [x86 or x86_64]
-c_compiler_version:  # [linux64]
-  - 7                # [linux64]
-cxx_compiler_version:  # [linux64]
-  - 7                # [linux64]
-c_compiler:    # [win]
-  - vs2019     # [win]
-cxx_compiler:  # [win]
-  - vs2019     # [win]
+
+# Need to downgrade compiler for nccl until it gets patched on conda forge
+# https://github.com/NVIDIA/nccl/issues/494#issuecomment-881074063
```

Comment:
Is that downgrade still necessary now that you picked the respective commit into the ctng-compiler feedstock?

Reply:
nvm, sorry, was reviewing commit by commit 🙈

Reply:
Where are you seeing this? You might have an out of date view. I forced push and cleand it up
```
$ cat conda_build_config.yaml
blas_impl:
  - mkl                        # [x86 or x86_64]
```